WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -186,6 +186,15 @@ final class ProductsViewController: UIViewController {
         if AppRatingManager.shared.shouldPromptForAppReview() {
             displayRatingPrompt()
         }
+
+        // Fix any incomplete animation of the refresh control
+        // when switching tabs mid-animation
+        refreshControl.resetAnimation(in: tableView) { [unowned self] in
+            // ghost animation is also removed after switching tabs
+            // show make sure it's displayed again
+            self.removePlaceholderProducts()
+            self.displayPlaceholderProducts()
```

Comment:
Probably I miss the context, but I'm not sure this is needed, since they are separate things. It should continue to work also without removing and re-adding the placeholder, maybe.

Reply:
I agree that this has nothing to do with the main issue - but as I was testing with the app I saw that when I triggered pull-to-refresh on Product List and Review List screens, the table views were displayed with placeholders; then when I switch to other tabs and back again, the animations of the placeholders were gone. 

This issue is not obvious if you have more than the maximum number of rows for the placeholder, however in my case I saw this: 
<img src="https://user-images.githubusercontent.com/5533851/124050940-b9d0ce80-da45-11eb-9ffc-8068f963e65a.png" width=400 />

The empty rows lost their animated overlay, and one single row for cached review showed up - which looks weird to me. Let me know what you think about this - thanks!

Reply:
(I should probably have included this in the description of the PR though) 😅